vcr/pe/presentation_definition.go

```diff
@@ -294,6 +294,11 @@ func matchFormat(format *PresentationDefinitionClaimFormatDesignations, credenti
 		return true
 	}
 
+	if credential.Format() == "" {
+		// holder credential
```

Comment:
what has holder credential (actually "issuer is holder"-credential) to do with format? Weird comment?

Reply:
Format is extracted from the proof. Holder credential has no Proof and thus no format.

Reply:
The credential can still be in either JWT (unsigned) or JSON-LD format?

Reply:
We have to rewrite some stuff. Let's hold that until VC data model 2 and it's new VC types. The two formats we're using now are gone in 202x?